C7GameData/CombatResult.cs

```diff
@@ -0,0 +1,22 @@
+namespace C7GameData
+{
+	public enum CombatResult {
```

Comment:
Is it worth differentiating here between killed & captured, or whether the attacker advances? This seems adequate for now.

Reply:
I hadn't thought about capturing units, but we'll probably want to add that as a possible result. Differentiating whether or not the attacker advances at this point would be awkward since CombatResults are produced by `MapUnit.fight`, which isn't responsible for determining whether or not the attacker can advance. That's done in `MapUnit.move` after it checks the result. If we never end up using CombatResult for anything other than the return value from `MapUnit.fight`, it should be renamed to FightResult for clarity.

Reply:
I think captured should be part of a future "Combat Part III".  It's an advanced rule, and IMO not as important as more basic things like adding technologies and buildings and map visibility.  Eventually, though, it probably should be a possible result.